content/en/platform/corda/5.0/key-concepts/cluster-admin/tech-stack/_index.md

```diff
@@ -45,7 +45,7 @@ Broadly speaking, there are two groups of databases:
 Corda creates one of each of the following types per virtual node:
 * `Vault` — contains the virtual node ledger data as well as data defined in CorDapp custom schemas.
 * `Crypto` — contains the virtual node (encrypted) cryptographic key material such as ledger keys.
-* `Uniqueness` (optional) — maintains a record of unspent and spent states generated as part of UTXO ledger transactions. This is only relevant for notary nodes.
+* `Uniqueness` (optional) — maintains a record of unspent and spent states generated as part of UTXO ledger transactions. This is only relevant for {{< tooltip >}}notary{{< /tooltip >}} nodes.
```

Comment:
tooltip for UTXO

Reply:
As per the PR comment, this will come in another PR.